components/InfoDialogsContent.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react'
+import InfoDialog from './InfoDialog'
+import ReadHereLink from './ReadHereLink'
+
+export const SplitsTypesInfo = () => (
+  <>
+    In music industry, there are two primary types of works:{' '}
+    <b>Musical Compositions</b> and{' '}
+    <b>
+      Sound Recordings <s>copyright and</s>
+    </b>{' '}
+    <b>
+      <s>publishing rights.</s>
+    </b>{' '}
+    These two kinds of works have their own copyright.
+    <ol className="list-decimal list-outside ml-6 mb-8">
+      <li>
+        <b>
+          Copyright in Musical Composition (Publishing Rights or Composition
+          Rights):
+        </b>{' '}
+        Relate to the “song” or underlying composition—the lyrics and melody of
+        a song, independent of any particular recording. These rights are
+        divided between the <b>songwriter</b> (or composer) and the{' '}
+        <b>music publisher.</b> The publisher manages the songwriter’s
+        composition by licensing it for use, collecting royalties, and ensuring
+        it is properly credited. The key components of publishing rights include{' '}
+        <b>performance rights, mechanical rights,</b> and synchronization rights
+        (for use in films, TV, etc.).
+      </li>
+      <li>
+        <b>Copyright in Sound Recordings (MASTER Rights):</b> Pertains to the
+        ownership of a particular recording of a performance of a song. Whoever
+        owns the master rights controls the use, distribution, reproduction, and
+        performance of that specific recording. These rights typically belong to
+        the record label or the artist who financed the production of the
+        recording, though they can be sold or licensed.
+      </li>
+    </ol>
+    Both rights are crucial for monetizing and legally protecting music. The
+    copyright controls the use of a specific recording, while publishing rights
+    control the use of the song&#39;s composition.
+  </>
+)
```

Comment:
Please keep each file to one component definition.
- actual: 2 components defined in one file.
- expected: one component defined per file.